storage/mysql/tree_storage.go

```diff
@@ -137,33 +137,27 @@ func expandPlaceholderSql(sql string, num int) string {
 }
 
 func decodeSignedTimestamp(signedEntryTimestampBytes []byte) (trillian.SignedEntryTimestamp, error) {
-	// TODO: This will probably switch to proto serialization later
 	var signedEntryTimestamp trillian.SignedEntryTimestamp
-	buffer := bytes.NewBuffer(signedEntryTimestampBytes)
-	dec := gob.NewDecoder(buffer)
-	err := dec.Decode(&signedEntryTimestamp)
+
+	err := proto.Unmarshal(signedEntryTimestampBytes, &signedEntryTimestamp)
 
 	if err != nil {
-		glog.Warningf("Failed to encode payload: %s", err)
+		glog.Warningf("Failed to decode payload: %s", err)
```

Comment:
Yeah it's because the original table column was payload, that still needs to be cleaned up. Done.